frontend/src/metabase/visualizations/echarts/pie/option/index.ts

```diff
@@ -0,0 +1,29 @@
+import type { EChartsOption } from "echarts";
+
+import type {
+  ComputedVisualizationSettings,
+  RenderingContext,
+} from "metabase/visualizations/types";
+
+import type { PieChartModel } from "../model/types";
+import { SUNBURST_SERIES_OPTIONS } from "./constants";
+
+export function getPieChartOption(
+  chartModel: PieChartModel,
+  settings: ComputedVisualizationSettings,
+  renderingContext: RenderingContext,
+): EChartsOption {
+  return {
+    textStyle: {
+      fontFamily: renderingContext.fontFamily,
+    },
+    series: {
+      ...SUNBURST_SERIES_OPTIONS,
```

Comment:
I'm intentionally leaving the options object sparse in this PR, my intention here is just setting the foundation. Later PRs will focus on adding specific parts of the object, e.g. show total setting, show percentages setting, etc